utility.lua

```diff
@@ -1835,10 +1835,12 @@ function Auxiliary.GetLinkCount(c)
 	else return 1 end
 end
 function Auxiliary.LCheckOtherMaterial(c,mg,lc)
-	local le=c:IsHasEffect(EFFECT_EXTRA_LINK_MATERIAL)
-	if not le then return true end
-	local f=le:GetValue()
-	return f(nil,lc,mg)
+	local le={c:IsHasEffect(EFFECT_EXTRA_LINK_MATERIAL)}
```

Comment:
if `le` is nil, it should return true here.

Reply:
https://github.com/Fluorohydride/ygopro-scripts/pull/1065/commits/9f594b9441cd171feeff01de5f44322a4ac57dce fixed